.github/codecov.yml

```diff
@@ -0,0 +1,3 @@
+codecov:
+  notify:
+    after_n_builds: 2
```

Comment:
Not sure if `2` is a good choice. If I understand correctly, whole checks run will make 4 codecov uploads. One per package tests per version (2), and one per docs tests per version (2).

Reply:
Right, I didn't take into consideration that the checks run for 2 versions.
However it works - the notification was pushed after all the checks have finished (at least that was how it looked).

Reply:
Actually, now that I've looked, the comment is edited a minute after it is created.